SampleApp/include/SampleApp/UIManager.h

```diff
@@ -403,6 +405,14 @@ class UIManager
      *  Prints audio injection failure message.
      */
     void printAudioInjectionFailureMessage();
+#ifdef XMOS_AVS_TESTS
+    /**
+     *  Return connection status
+     */
+    avsCommon::sdkInterfaces::ConnectionStatusObserverInterface::Status getConnectionStatus() {
```

Comment:
I suggest putting `const` after the empty parameter list and before the opening brace, i.e., `getConnectionStatus() const {`, since the function doesn't alter the value of any data member.  Doing so allows the compiler to check that no data members have their value altered by the function call.